window/src/os/windows/window.rs

```diff
@@ -947,6 +1180,23 @@ unsafe fn wm_windowposchanged(
     Some(0)
 }
 
+unsafe fn wm_sizeraw(hwnd: HWND, _msg: UINT, wparam: WPARAM, _lparam: LPARAM) -> Option<LRESULT> {
```

Comment:
I wonder if we can eliminate this?

`inner.last_resize_type` is effectively the same thing as `inner.appearance`, so I think it would be nice to remove that redundancy, but if we can't it would be good to have some comments in the code that explain the nuance.

If we can remove tracking `last_resize_type`, then the change I suggested for `TermWindow::resize` would allow removing the dispatch of `AppearanceChanged` here as well.